_data/icons/peculiar.json

```diff
@@ -0,0 +1,8 @@
+[
+    {
+	"url":"ipfs://QmdReVrFF3GRRG3GvrB3E7GZ6yNf7fJ7zNBmCbQWydP8f5",
```

Comment:
where did you pin this CID? I cannot resolve it

Reply:
I reupload this logo to ipfs. It can be accessed at https://ipfs.io/ipfs/bafkreieafwsqb6dr47hos4fzorwgxly6thjlapzzgogonswykfqfbtoipq